carma/launch/carma_src.launch

```diff
@@ -47,22 +46,10 @@
   <arg name="simulation_mode" default="false" doc="True if CARMA Platform is launched with CARLA Simulator"/>
   
   <!-- Debug -->
```

Comment:
Might as well remove this comment as well since it doesn't apply to anything now